index.html

```diff
@@ -0,0 +1,346 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <link rel='stylesheet' type='text/css' href='style.css'>
+  <meta charset="utf-8">
+  <title>Ghost Host Homepage</title>
+</head>
+  
+<main>
+     <header id = "home" class="#">
```

Comment:
header should not be inside main as per https://www.w3schools.com/tags/tag_main.asp